burger_war/MarkerStatus[] markers